doc/source/new/auditlogging.rst

```diff
@@ -450,37 +451,45 @@ The ``auditlogviewer`` tool is used to dump audit logs. Run the ``auditlogviewer
- existing keyspace "auditlogkeyspace"
- Type: AuditLog
- LogMessage:
- user:anonymous|host:10.0.2.238:7000|source:/127.0.0.1|port:46264|timestamp:1564714690968|type :DROP_KEYSPACE|category:DDL|ks:auditlogkeyspace|operation:DROP KEYSPACE AuditLogKeyspace;
- Type: AuditLog
- LogMessage:
- user:anonymous|host:10.0.2.238:7000|source:/3.91.56.164|port:42406|timestamp:1564714708329|ty pe:CREATE_KEYSPACE|category:DDL|ks:auditlogkeyspace|operation:CREATE KEYSPACE
- AuditLogKeyspace
- WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 1};
- Type: AuditLog
- LogMessage:
- user:anonymous|host:10.0.2.238:7000|source:/127.0.0.1|port:46264|timestamp:1564714870678|type :USE_KEYSPACE|category:OTHER|ks:auditlogkeyspace|operation:USE auditlogkeyspace;
- [ec2-user@ip-10-0-2-238 hourly]$
+ Type: audit
+  LogMessage:
+  user:anonymous|host:10.0.2.238:7000|source:/127.0.0.1|port:46264|timestamp:1564711427328|type :USE_KEYSPACE|category:OTHER|ks:auditlogkeyspace|operation:USE AuditLogKeyspace;
+  Type: audit
+  LogMessage:
+  user:anonymous|host:10.0.2.238:7000|source:/127.0.0.1|port:46264|timestamp:1564711427329|type :USE_KEYSPACE|category:OTHER|ks:auditlogkeyspace|operation:USE "auditlogkeyspace"
+  Type: audit
+  LogMessage:
+  user:anonymous|host:10.0.2.238:7000|source:/127.0.0.1|port:46264|timestamp:1564711446279|type :SELECT|category:QUERY|ks:auditlogkeyspace|scope:t|operation:SELECT * FROM t;
+  Type: audit
+  LogMessage:
+  user:anonymous|host:10.0.2.238:7000|source:/127.0.0.1|port:46264|timestamp:1564713878834|type :DROP_TABLE|category:DDL|ks:auditlogkeyspace|scope:t|operation:DROP TABLE IF EXISTS
+  AuditLogKeyspace.t;
+  Type: audit
+  LogMessage:
+  user:anonymous|host:10.0.2.238:7000|source:/3.91.56.164|port:42382|timestamp:1564714618360|ty
+  pe:REQUEST_FAILURE|category:ERROR|operation:CREATE KEYSPACE AuditLogKeyspace
+  WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 1};; Cannot add
+  existing keyspace "auditlogkeyspace"
+  Type: audit
+  LogMessage:
+  user:anonymous|host:10.0.2.238:7000|source:/127.0.0.1|port:46264|timestamp:1564714690968|type :DROP_KEYSPACE|category:DDL|ks:auditlogkeyspace|operation:DROP KEYSPACE AuditLogKeyspace;
+  Type: audit
+  LogMessage:
+  user:anonymous|host:10.0.2.238:7000|source:/3.91.56.164|port:42406|timestamp:1564714708329|ty pe:CREATE_KEYSPACE|category:DDL|ks:auditlogkeyspace|operation:CREATE KEYSPACE
+  AuditLogKeyspace
+  WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 1};
+  Type: audit
+  LogMessage:
+  user:anonymous|host:10.0.2.238:7000|source:/127.0.0.1|port:46264|timestamp:1564714870678|type :USE_KEYSPACE|category:OTHER|ks:auditlogkeyspace|operation:USE auditlogkeyspace;
+  [ec2-user@ip-10-0-2-238 hourly]$
+  Type: audit
+  LogMessage: user:cassandra|host:localhost/127.0.0.1:7000|source:/127.0.0.1|port:65282|timestamp:1622630496708|type:CREATE_ROLE|category:DCL|operation:create role role1 WITH password = '*******';
```

Comment:
maybe for the purpose of the documentation but they are recorded as they were typed in cqlsh so if you enter it lowercase, it will be lowercased in the log too.

Reply:
changed to uppercase, resolving.